content/en/th3/500_accessibility/311_screenReaders.md

```diff
@@ -7,28 +7,64 @@ weight: '502'
 
 ## Reading with a screen reader
 
-When launching Thorium, if a screen reader is detected, the relative
-functions are activated.
+### Activating screen reader support 
 
-Note that screen readers use a separate buffer to access the HTML
-content which results in the visual scroll / page offset to be out of
-sync. In other words, Thorium is generally not aware of the user\'s
+Thorium automatically detects assistive technology, but, since 3.3, users must explicitly activate screen reader support by ticking the checkbox in the application settings (this is a global preference for the main library window as well as all reader windows). 
+
+In the previous versions, this support was automatically activated. We had to change because of third party apps creating false positives because they activate assistive technology privileges.
+
+Thorium adapts its behavior when screen reader support is enabled:
+* Text input fields for bookshelf searches require explicit validation (e.g., pressing Enter or clicking the search button) to trigger the search, instead of the default "search as you type" behavior.
+* In the reader window, EPUB HTML documents default to scroll mode instead of paginated presentation to avoid issues with CSS columns.
+* HTML document webviews are fully refreshed on navigation, instead of being reused, to prevent screen readers from losing track of the text.
+* Detection of the user's reading location based on visual scroll offset is disabled to avoid misinterpretations caused by screen reader interaction-induced HTML layout shifts.
+
+### Reading position synchronisation
+
+Since Thorium 3.2, Thorium knows the screen reader reading position and therefore maintains visual scroll synchronisation. In previous versions, because of the screen reader using a separate buffer, 
```

Comment:
Mhmm, that's not true. Thorium needs the user to interact with the text (i.e. click, select) in order estimate the current reading location.

Reply:
deleting the whole section as it is now explained at the last bullet point of the previous section.

Reply:
> Confused, thought that was fixed

I think NVDA improved their support for native DOM Range selections.